.eslintrc

```diff
@@ -33,7 +33,10 @@
     "sandbox": true,
     "context": false,
     "global": false,
-    "describes": true
+    "describes": true,
+    "require": false,
```

Comment:
Please move these into a sub folder's `.eslintrc`. They're dangerous in the `src` folder.

Reply:
Good point. I was about to ask. Done.